[NOW RESOLVED] ~~The CI fails in this branch but also on main:~~
```
Directory 'actions-ci' does not exist!
CI expects the 'actions-ci-circuitpython-libs' tools being checked out to said directory.
Error: Process completed with exit code 1.
```

--- a/.github/workflows/githubci.yml
+++ b/.github/workflows/githubci.yml
@@ -21,4 +21,5 @@ jobs:
         architecture: 'x64' # optional x64 or x86. Defaults to x64 if not specified
     - name: pre-install
       run: |
+        touch requirements.txt # Empty requirements expected by install.sh
         source actions-ci/install.sh

--- a/.github/workflows/githubci.yml
+++ b/.github/workflows/githubci.yml
@@ -13,10 +13,12 @@ jobs:
 
     steps:
     - uses: actions/checkout@v3
+      with:
+        path: "actions-ci"
     - uses: actions/setup-python@v4
       with:
         python-version: '3.x' # Version range or exact version of a Python version to use, using semvers version range syntax.
         architecture: 'x64' # optional x64 or x86. Defaults to x64 if not specified
     - name: pre-install
       run: |
-        source install.sh
+        source actions-ci/install.sh

--- a/.github/workflows/githubci.yml
+++ b/.github/workflows/githubci.yml
@@ -12,10 +12,10 @@ jobs:
     runs-on: ubuntu-latest
 
     steps:
-    - uses: actions/checkout@v3
+    - uses: actions/checkout@v4
       with:
         path: "actions-ci"
-    - uses: actions/setup-python@v4
+    - uses: actions/setup-python@v5
       with:
         python-version: '3.x' # Version range or exact version of a Python version to use, using semvers version range syntax.
         architecture: 'x64' # optional x64 or x86. Defaults to x64 if not specified

